### PR TITLE
Add support for DDM18SD (noname) single phase meters

### PR DESCRIPTION
**Pull request by merbanan**

### CODE REVIEW

**Comment by andig** [comment]
Could you run

    mbmd doc

too?

File: meters/rs485/ddm18sd.go [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+const (
+	METERTYPE_DDM = "DDM"
+)
+
+type DDMProducer struct {
+	Opcodes
+}
+
+func NewDDMProducer() Producer {
+
```
Comment (andig) [inline meters/rs485/ddm18sd.go:18]: Remove empty line

**Comment by merbanan** [comment]
I also have tools to configure the sdm120m, or-we-514 and the ddm18sd. Maybe that is worth a link somewhere?

https://github.com/merbanan/modbus_cli_tools

It lacks in the documentation department but you can configure most device parameters with it. And you can use serial devies, modbus tcp and ethernet to 485 bridges.

File: meters/rs485/ddm18sd.go (hunk 1 above)
Comment (merbanan) [reply meters/rs485/ddm18sd.go]: Fixed

**Comment by merbanan** [comment]
Scanning will result in a false positive for SDM.

2021/08/29 23:52:19 * #203 type SDM (Model: Eastron SDM630)

Probing could be better but I don't think is would be worth it.

**Pull request merged**